pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -201,10 +212,24 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 	// DELETE: the CR was deleted
 	if !cephFilesystemSubVolumeGroup.GetDeletionTimestamp().IsZero() {
 		logger.Debugf("deleting subvolume group %q", namespacedName)
+
+		cephFsSvgList := &cephv1.CephFilesystemSubVolumeGroupList{}
+		// List cephFilesystemSubvolumeGroup CR based on filesystem and spec.name
+		matchingKey := fmt.Sprintf("%s/%s", getSubvolumeGroupName(cephFilesystemSubVolumeGroup), cephFilesystemSubVolumeGroup.Spec.FilesystemName)
+		err = r.client.List(r.opManagerContext, cephFsSvgList, &client.MatchingFields{cephSVGFileSystemNameIndex: matchingKey})
```

Comment:
will this list in the namespace where SVG is present or in all namespaces? i dont see namespace level listing

Reply:
This will do all namespace listing

Reply:
yes that will be a corner case problem. I'll add another if check matching cephcluster namespace and SVG namespace as client list is done per cluster  wdyt?

Reply:
done